ansible/idr-s3gateway.yml

```diff
@@ -28,17 +28,41 @@
       nginx_proxy_http2: true
       nginx_proxy_force_ssl: true
       # nginx_proxy_hsts_age: 31536000
-      nginx_proxy_backends:
-        - name: s3gateway
-          location: "~ ^/{{ idr_minio_s3_gateway_bucket | default('example') }}(/.*)?$"
-          server: "http://localhost:9000"
-      # Disable buffering and allow unlimited file sizes
-      # https://serverfault.com/a/818090
-      nginx_proxy_additional_directives:
-        - client_max_body_size 0
-        - proxy_http_version 1.1
-        - proxy_request_buffering off
-        - proxy_buffering off
+      nginx_proxy_sites:
+        # idr-ftp.openmicroscopy.org: S3 read-write /idr-upload only
```

Comment:
Not specific to this PR but `idr-ftp...` for s3 seems inflammatory ;)

Reply:
No one came up with a better name 😄 .

Though now that we have idr-s3 I think we could put it under there, providing we never need to split the two buckets onto different servers.

Reply:
Although we extensively discussed the `idr-s3` proxy server for the public Zarr data, I do not remember discussing the proxy server name for the `idr-upload` bucket. I had been probably wrongly `idr-s3` could be reused as well.

I also feel `idr-ftp.openmicroscopy.org/idr-upload` is confusing for submitters esp. if we keep maintaining a FTP service in parallel. As the scope of this PR is the public read-only `idr` bucket, I would not raise as a blocker but let's not sidestep this discussion before we start pointing submissions at the upload bucket.

Reply:
https://github.com/IDR/deployment/issues/270